packages/subscript/Subscript.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var PropertyAnnotation = require('../../model/PropertyAnnotation');
+
+var Subscript = PropertyAnnotation.extend({
+  name: 'subscript'
```

Comment:
When we stick with Class.extend, we should remove any custom magic.
I.e. this should go into a static block.